### PR TITLE
Closes #1 - Create initial index.html markup

### LINKED ISSUE #1: Create initial index.html markup

### PR DESCRIPTION
**Pull request by koldovsky**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Document</title>
+  </head>
+  <body></body>
```
Comment (YuriiSynopol) [inline index.html]: Please close body in second row

Comment (Orest-Horak) [inline index.html]: please close body in a second row

File: index.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+</html>
```
Comment (YarkoLight) [inline index.html:12]: please close body in second row

File: index.html (hunk 1 above)
Comment (X1rus) [inline index.html]: please close body in second row

**Review by Orest-Horak — CHANGES_REQUESTED** [review]

**Review by YuriiSynopol — CHANGES_REQUESTED** [review]

**Review by X1rus — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (TarasSkobalo) [inline index.html]: please close body in second row

**Review by TarasSkobalo — CHANGES_REQUESTED** [review]

**Review by Orest-Horak — APPROVED** [review]

**Review by YuriiSynopol — APPROVED** [review]

**Review by X1rus — APPROVED** [review]

**Review by TarasSkobalo — APPROVED** [review]

**Review by YarkoLight — APPROVED** [review]

**Pull request merged**